packages/telemetry/src/telemetryChildProcess.ts

```diff
@@ -7,6 +7,14 @@ import { TelemetryReporter } from './telemetryReporter';
   const options = JSON.parse(args[0]);
   const reporter = await TelemetryReporter.create(options);
   process.on('message', event => {
-    reporter.sendTelemetryEvent(event.eventName, event.attributes);
+    if (event.eventName) {
```

Comment:
I think it would be a cleaner implementation if you add a flag to the event that dictates which method is used. For example, 

```
{ type: 'trace' }
{ type: 'exception' }
{ type: 'event' }
etc...
```

Reply:
@mdonnalley Take a look at the changes in `fix: flag type when passing data to child process`